src/test/java/org/sasanlabs/service/vulnerability/ssrf/SSRFVulnerabilityTest.java

```diff
@@ -0,0 +1,166 @@
+package org.sasanlabs.service.vulnerability.ssrf;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Collections;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.io.TempDir;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.sasanlabs.service.vulnerability.bean.GenericVulnerabilityResponseBean;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+class SSRFVulnerabilityTest {
+
+    private static final String INVALID_URL = "invalidUrl";
+    private static final String INVALID_URL_MESSAGE = "Provided URL not valid";
+    private static final String TEMP_FILE_CONTENT = "Temp file content";
+    private static final String METADATA_URL_AWS = "http://169.254.169.254/1.0";
+    private static final String METADATA_URL_OTHER = "http://[::ffff:169.254.169.254]/1.0";
+    private static final String METADATA_URL_CONTENT = "meta-data";
+    private static final String OTHER_URL =
```

Comment:
instead of using a custom gist url, please use the project url instead or OWASP github url or owasp.org as chances of that changing are very less in comparison to this gist.

Reply:
Updated to project url.